rust/scx_utils/build.rs

```diff
@@ -3,8 +3,13 @@
 // This software may be used and distributed according to the terms of the
 // GNU General Public License version 2.
 
+use vergen::EmitBuilder;
 include!("src/builder.rs");
 
 fn main() {
-    Builder::new().build()
+    Builder::new().build();
+    EmitBuilder::builder()
+        .all_git()
```

Comment:
One issue that I see here is when we build this from a source package we won't have `.git` and apparently in this case vergen would emit something like `VERGEN_IDEMPOTENT_OUTPUT`. It'd be nice if we could fallback to the version define in Cargo.toml in this case...

Reply:
`.cargo_target_triple()` can be useful too.